Feature/add namespace (#13)
banzaicloud/terraform-provider-k8s: README.md

This project is a maintained fork of [ericchiang/terraform-provider-k8s](https://github.com/ericchiang/terraform-provider-k8s).

## Installation

### The Go Get way

Use `go get` to install the provider:

```
go get -u github.com/banzaicloud/terraform-provider-k8s
```

> Register the plugin in `~/.terraformrc` (see [Documentation]( https://www.terraform.io/docs/commands/cli-config.html ) for Windows users): 

Reviewer: ```suggestion
Register the plugin in `~/.terraformrc` (see [Documentation](https://www.terraform.io/docs/commands/cli-config.html) for Windows users): 
```

```hcl
providers {
  k8s = "/$GOPATH/bin/terraform-provider-k8s"
}
```

### The Terraform Plugin way  (enable versioning)

> Download a release from the [Release page]( https://github.com/banzaicloud/terraform-provider-k8s/releases ) and make sure the name matches the following convention

Reviewer: ```suggestion
Download a release from the [Release page](https://github.com/banzaicloud/terraform-provider-k8s/releases) and make sure the name matches the following convention:
```

Author: also in link with my comment on issue #6

> | OS      | version | Name                              |

Reviewer: ```suggestion
| OS      | Version | Name                              |
```

| ------- | ------- | --------------------------------- |

> | LINUX   | 0.4.0   | terraform-provider-eks_v0.4.0     |

Reviewer: I think it  would make sense if these names were real (using `k8s`)

Author: Also in link with my comment on issue #6

|         | 0.3.0   | terraform-provider-eks_v0.3.0     |
| Windows | 0.4.0   | terraform-provider-eks_v0.4.0.exe |
|         | 0.3.0   | terraform-provider-eks_v0.3.0.exe |

> Install the plugin using [Terraform Third-party Plugin Documentation]( https://www.terraform.io/docs/configuration/providers.html#third-party-plugins )

Reviewer: ```suggestion
Install the plugin using [Terraform Third-party Plugin Documentation](https://www.terraform.io/docs/configuration/providers.html#third-party-plugins):
```

resource "k8s_manifest" "nginx-deployment" {
  content = "${data.template_file.nginx-deployment.rendered}"
}

> # creating a secondary ressource in the nginx namespace

Reviewer: ```suggestion
# creating a second resource in the nginx namespace
```

No resources found.
```

> 

Reviewer: ```suggestion
```

> ## Helm workflow

Reviewer: What does this section add to the provider itself?
